### PR TITLE
Initial import

### PR DESCRIPTION
**Pull request by jmtd**
No CCT section: no scripts to execute

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: "jboss-decisionserver-6/decisionserver63-openshift"
+version: "1.5"
```
Comment (goldmann) [inline image.yaml:2]: Add `description`.

File: image.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      value: "true"
+      description: "The BRP capability is disabled in this image."
+    - name: "JBOSS_PRODUCT"
+      value: "decisionserver"
+    - name: "JBOSS_DECISIONSERVER_VERSION"
+      value: "6.3.4.GA"
+    - name: "PRODUCT_VERSION"
+      value: "6.3.4.GA"
+dogen:
+    version: 2.1.0
```
Comment (goldmann) [inline image.yaml:30]: dist-gir repo is missing:

```
repo: jboss-decisionserver-6-docker
branch: ce-1.5-openshift-decisionserver-6.3-jdk-8-rhel-7
```

**Review by goldmann — CHANGES_REQUESTED** [review]

File: image.yaml (hunk 1 above)
Comment (jmtd) [reply image.yaml:2]: It's two lines lower :)

Comment (goldmann) [reply image.yaml:2]: Ooooh. Sorry!

**Review by goldmann — APPROVED** [review]

**Pull request merged**